src/app/actions/equipment/a_quantity_add.ts

```diff
@@ -0,0 +1,54 @@
+import { ApiService } from "src/app/shared/services/apiService/api.service";
+import { ViewService } from "src/app/shared/services/viewService/view.service";
+import { WizardService } from "src/app/shared/services/wizardService/wizard.service";
+import { switchMap, filter, tap, defaultIfEmpty, of } from "rxjs";
+import { CID } from "src/app/shared/config/constants.json";
+import { HttpParams } from "@angular/common/http";
+import { Equipment } from "src/app/shared/models/apiService.model";
+
+export function a_equipment_add_quantity(vs: ViewService, apis: ApiService, ws: WizardService) {
+  return of(vs.objID).pipe(
+    filter(objID => objID !== null),
+    switchMap(() => {
```

Comment:
It'll look better as 
```suggestion
   switchMap(() => ws.create('addQuantity')),
```